### ISSUE TITLE
Homebridge 1.3.0 reports a warning for the characteristic 'On': SET handler

### ISSUE THREAD

**Issue by jsiegenthaler** [Sunoo/homebridge-camera-ffmpeg#1045]
**Describe The Problem:**

Upgraded to Homebridge v1.3.0 today.
Noted that Homebridge is reporting a warning for homebridge-camera-ffmpeg 

**To Reproduce:**

1. Ensure Homebridge v1.3.0 is installed
2. View Homebridge logs

**Logs:**

```
[20/02/2021, 12:52:12] [homebridge-camera-ffmpeg] This plugin generated a warning from the characteristic 'On': SET handler returned write response value, though the characteristic doesn't support write response! See https://git.io/JtMGR for more info.
```

**Environment:**

* **Node.js Version**: v14.15.5
* **NPM Version**: v6.14.11
* **Homebridge Version**: v1.3.0
* **Homebridge Camera FFmpeg Version**: v3.0.6
* **Homebridge Config UI X Plugin Version**: v4.39.0 
* **Operating System**: Raspbian GNU/Linux Buster (10)

**Comment by Sunoo** [comment]
Please share your config so I can narrow down where that’s coming from.

**Comment by jsiegenthaler** [comment]
@Sunoo 

It's a new warning that appears in Homebridge v1.3.0
Make sure you are on v1.3.0 and then try a config like my config:

Here's my config:

```
        {
            "name": "Camera FFmpeg",
            "porthttp": 8080,
            "cameras": [
                {
                    "name": "Lounge",
                    "manufacturer": "Logitech Inc.",
                    "model": "QuickCam IM/Connect",
                    "serialNumber": "1234",
                    "firmwareRevision": "1.0",
                    "motion": true,
                    "doorbell": true,
                    "switches": true,
                    "motionTimeout": 1,
                    "videoConfig": {
                        "source": "-f video4linux2 -r 30 -s 640x480 -i /dev/video0",
                        "maxWidthold": 640,
                        "maxHeightold": 480,
                        "maxFPSold": 25,
                        "vcodec": "libx264",
                        "audio": false,
                        "debug": false
                    }
                },
                {
                    "name": "Front Door",
                    "manufacturer": "Me",
                    "model": "Camera-ffmpeg Video Doorbell",
                    "serialNumber": "1234",
                    "firmwareRevision": "1.0",
                    "motion": false,
                    "doorbell": true,
                    "switches": true,
                    "motionTimeout": 1,
                    "videoConfig": {
                        "source": "-re -ignore_loop 0 -i file:///home/pi/Public/fuo.gif",
                        "maxStreams": 2,
                        "maxWidth": 1280,
                        "maxHeight": 720,
                        "maxFPS": 10,
                        "maxBitrate": 300,
                        "vcodec": "libx264",
                        "packetSize": 1316,
                        "audio": false,
                        "debug": true
                    }
                }
            ],
            "platform": "Camera-ffmpeg"
        },
```

**Comment by Sunoo** [comment]
I think I found it, looks like it should only happen when you toggle one of the switches though?

**Comment by jsiegenthaler** [comment]
I wasn't doing anything apart from restarting Homebridge (updated today to v1.3.0).
 
Maybe HomeKit was refreshing the status with a GET? 
I suggest you give your code a good thorough test on HomeKit v1.3.0, as it shows you warnings for any HomeKit GET/SET mistakes in your code... I found two in mine ;)

**Comment by Sunoo** [comment]
Fixed in 3.1.0

**Comment by jsiegenthaler** [comment]
Thanks @Sunoo
